### PR TITLE
SideDrawer Components

### PR DESCRIPTION
**Pull request by fares98**
Create a components for the sidebar

### CODE REVIEW

File: client/src/components/common/SideDrawer/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const SideDrawer = ({ loggedIN }) => {
+  const notLogedArr = ['Home', 'Services', 'Sign Up', 'Log In'];
+  const logedArr = [
```
Comment (FadiAlamassi) [inline client/src/components/common/SideDrawer/index.js]: fix typos

File: client/src/components/common/DrawerToggleButton/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const DrawerToggleButton = ({ clicked, className }) => {
+  return (
+    <article className={className} onClick={clicked}>
```
Comment (AbdallahAmmar96) [inline client/src/components/common/DrawerToggleButton/index.js]: Read more about the article tag 
[here](https://www.w3schools.com/tags/tag_article.asp)

File: client/src/components/common/DrawerToggleButton/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const DrawerToggleButton = ({ clicked, className }) => {
```
Comment (AbdallahAmmar96) [inline client/src/components/common/DrawerToggleButton/index.js]:
```suggestion
const DrawerToggleButton = ({ onClick, className }) => {
```

File: client/src/components/common/DrawerToggleButton/index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <article className={className} onClick={clicked}>
+      <span className="first"></span>
+      <span className="second"></span>
```
Comment (AbdallahAmmar96) [inline client/src/components/common/DrawerToggleButton/index.js]: The three classes are too common, try to use BEM
[BEM Link](http://getbem.com/naming/)

File: client/src/components/common/SideDrawer/index.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+import './style.css';
+
+const SideDrawer = ({ loggedIN, className }) => {
+  const notLogedArr = ['Home', 'Services', 'Sign Up', 'Log In'];
+  const logedArr = [
+    <img
+      src="https://png.pngtree.com/element_our/png/20181206/users-vector-icon-png_260862.jpg"
+      alt="UserImage"
+      className="img"
```
Comment (AbdallahAmmar96) [inline client/src/components/common/SideDrawer/index.js]: Same thing here for the class name

File: client/src/components/common/SideDrawer/index.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const SideDrawer = ({ loggedIN, className }) => {
+  const notLogedArr = ['Home', 'Services', 'Sign Up', 'Log In'];
```
Comment (AbdallahAmmar96) [inline client/src/components/common/SideDrawer/index.js]:
```suggestion
  const notLoggedItems = ['Home', 'Services', 'Sign Up', 'Log In'];
```

File: client/src/components/common/SideDrawer/index.js [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  const logedArr = [
```
Comment (AbdallahAmmar96) [inline client/src/components/common/SideDrawer/index.js]:
```suggestion
  const loggedItems = [
```

File: client/src/components/common/SideDrawer/index.js [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    'home',
+    'services',
+    'profile',
+    'messages',
+    'jobs',
+    'logOut',
+  ];
+
+  const items = arr =>
+    arr.map(ele => (
```
Comment (AbdallahAmmar96) [inline client/src/components/common/SideDrawer/index.js]:
```suggestion
    arr.map(element => (
```
or item

File: client/src/components/common/SideDrawer/index.js [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+    />,
+    'home',
+    'services',
+    'profile',
+    'messages',
+    'jobs',
+    'logOut',
+  ];
+
+  const items = arr =>
```
Comment (AbdallahAmmar96) [inline client/src/components/common/SideDrawer/index.js]:
```suggestion
  const items = array =>
```

Comment (AbdallahAmmar96) [inline client/src/components/common/SideDrawer/index.js]: Make the function name more readable

File: client/src/components/common/SideDrawer/style.css [hunk 10]
```diff
@@ -0,0 +1,59 @@
+a{
```
Comment (AbdallahAmmar96) [inline client/src/components/common/SideDrawer/style.css]: Put this common style in the App/style.css

**Pull request merged**